assets/src/infrastructure/ui/layout/modal/styles.scss

```diff
@@ -1,23 +1,41 @@
 @import 'application/ui/styles/mixins/breakpoints.scss';
 
-.ee-modal .ee-modal {
-	&__header {
-		&.ee-modal__header {
-			[dir='rtl'] & {
-				padding-right: var(--ee-padding-extreme);
+.ee-modal {
+	.ee-modal {
+		&__header {
+			&.ee-modal__header {
+				[dir='rtl'] & {
+					padding-right: var(--ee-padding-extreme);
+				}
+
+				@include max782px {
+					padding-left: var(--ee-padding-smaller);
+					padding-right: var(--ee-padding-extreme);
+				}
 			}
+		}
 
+		&__body {
 			@include max782px {
 				padding-left: var(--ee-padding-smaller);
-				padding-right: var(--ee-padding-extreme);
+				padding-right: var(--ee-padding-smaller);
 			}
 		}
 	}
 
-	&__body {
-		@include max782px {
-			padding-left: var(--ee-padding-smaller);
-			padding-right: var(--ee-padding-smaller);
+	@at-root [role='alertdialog']#{&} {
+		.ee-confirm-close {
+			display: inline-flex;
+			position: absolute;
+			top: 8px;
+			right: 12px;
+			width: var(--ee-padding-big);
+			height: var(--ee-padding-big);
+			svg {
+				height: var(--ee-border-radius-big);
+				margin: 0;
+				width: var(--ee-border-radius-big);
+			}
```

Comment:
the change in this file is only related to this block being moved here from that modal close btn